### PR TITLE
REFACTOR: [okx] refactor book and kline

### PR DESCRIPTION
**Pull request by bailantaotao**
K Line is currently not functioning because it is required to subscribe to another URL: wss://ws.okx.com:8443/ws/v5/business.

So, I simply refactor it.

### CODE REVIEW

File: pkg/exchange/okex/parse.go [hunk 1]
```diff
@@ -15,288 +14,329 @@ import (
...
+		}
 	}
 
 	return book
 }
 
-type BookEntry struct {
-	Price         fixedpoint.Value
-	Volume        fixedpoint.Value
+type PriceVolumeOrder struct {
```
Comment (c9s) [inline pkg/exchange/okex/parse.go:162]: we can embed types.PriceVolume here?

**Review by c9s — APPROVED** [review]

File: pkg/exchange/okex/parse.go (hunk 1 above)
Comment (bailantaotao) [reply pkg/exchange/okex/parse.go:162]: OK

**Comment by c9s** [comment]
build failed

**Comment by narumiruna** [comment]
rebase

**Pull request merged**